alwaysonline.py

```
@@ -111,6 +112,30 @@ def fetch_origin_server(url: str):
 
 
+def query_to_serp(url: str):
+    try:
+        # Process both removal of http:// or https:// and replacement of special characters at once
+        # ^https?:\/\/ removes http:// or https://, [^\w\s] removes special characters
+        q = re.sub(r'^https?:\/\/|[^\w\s]', ' ', url)
+
+        url = "%s/api.php?q=%s" % (librey_url, q)
+        response = requests.get(url, verify=False)
+        if response.status_code != 200:
+            return 502, str(e).encode(client_encoding)
```

Ruff

Undefined name `e`

(F821)